src/cobra/core/dictlist.py

```diff
@@ -37,32 +38,33 @@ def __init__(self, *args):
             else:
                 self.extend(other)
 
-    def has_id(self, id):
+    def has_id(self, id: Any) -> bool:
```

Comment:
Pretty sure `id` has to be a string, no?
```suggestion
    def has_id(self, id: str) -> bool:
```

Reply:
Or is it `Union[cobra.core.Object, str]`?